susemanager-utils/testing/automation/java-checkstyle.sh

```diff
@@ -41,5 +41,6 @@ CHOWN_CMD="/manager/susemanager-utils/testing/automation/chown-objects.sh $(id -
 
 docker pull $REGISTRY/$PGSQL_CONTAINER
 docker run --privileged --rm=true -v "$GITROOT:/manager" \
+    -v "${HOME}/.obs-maven-cache:/manager/java/.obs-to-maven-cache" \
```

Comment:
Is there a reason why it is not named `${HOME}/.obs-to-maven-cache`, wouldn't that be more standardized?

Reply:
tiredness at the end of a long day? I'll fix it